lib/MySQL_Thread.cpp

```diff
@@ -1142,7 +1141,6 @@ MySQL_Threads_Handler::MySQL_Threads_Handler() {
 	variables.poll_timeout_on_failure=100;
 	variables.have_compress=true;
 	variables.have_ssl = false; // disable by default for performance reason
-	variables.client_found_rows=true;
```

Comment:
Maybe the member MySQL_Threads_Handler::variables::client_found_rows in MySQL_Thread.h should be removed as well?

Reply:
@jaredev not only that, but also the thread local variables definitions should be removed. Thanks for the heads up, fixed.